README.md

```diff
@@ -1,5 +1,6 @@
 # MarlinReco
 [![Build Status](https://travis-ci.org/iLCSoft/MarlinReco.svg?branch=master)](https://travis-ci.org/iLCSoft/MarlinReco)
```

Comment:
Can you also remove this badge then? :-)

Reply:
Done. :)